### PR TITLE
IGNITE-13606

### PR DESCRIPTION
**Pull request by Nikita-tech-writer**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: docs/_docs/monitoring-metrics/metrics.adoc [hunk 1]
```diff
@@ -24,6 +24,23 @@ You can also access them programmatically.
 
 On this page, we've collected the most useful metrics and grouped them into various common categories based on the monitoring task.
 
+[NOTE]
```
Comment (dmagda) [inline docs/_docs/monitoring-metrics/metrics.adoc:27]: How about having this as a chapter? That's an essential configuration step you can't miss.

== Enabling JMX for Ignite

{the content you have}

File: docs/_docs/monitoring-metrics/metrics.adoc [hunk 2]
```diff
@@ -24,6 +24,23 @@ You can also access them programmatically.
...
+By default, the JMX automatic configuration is disabled.
+To enable it, configure the following environment variables:
+
+* For `control.sh`, configure the `CONTROL_JVM_OPTS` variable
+* For `ignite.sh`, configure the `JVM_OPTS` variable
+
+For example:
+[source,shell]
+----
+JVM_OPTS="-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=${JMX_PORT} \
```
Comment (dmagda) [inline docs/_docs/monitoring-metrics/metrics.adoc:38]: Have a complete instruction for both Unix and Windows. For instance, "ignite.sh ... parameters"

Comment (Nikita-tech-writer) [reply docs/_docs/monitoring-metrics/metrics.adoc:38]: Not sure whether it's necessary to add separate tabs for the environment variable configuration (I'm pretty sure that it does not depend on the OS type).

Comment (Nikita-tech-writer) [reply docs/_docs/monitoring-metrics/metrics.adoc:39]: Not sure whether it's necessary to add separate tabs for the environment variable configuration.
As for the CONTROL_JVM_OPTS variable, its configuration depends on the Java version, so we can't provide a universal example here.

**Comment by Nikita-tech-writer** [comment]
@Mmuzaf could you please cherry-pick this to 2.10?

**Pull request merged**